src/mmrelay/db_utils.py

```
@@ -771,24 +836,128 @@ def _store(cursor: sqlite3.Cursor) -> None:
+        current_ids (set[str]): Set of Meshtastic node IDs that should be kept.
+
+    Returns:
+        int: Number of rows deleted.
+
+    Raises:
+        ValueError: If `table` is not a supported names table.
+    """
+    select_sql = _SELECT_STALE_IDS_SQL_BY_TABLE.get(table)
+    delete_sql = _DELETE_STALE_ID_SQL_BY_TABLE.get(table)
+    if select_sql is None or delete_sql is None:
+        raise ValueError(f"Invalid table name: {table}")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)